bedrock/firefox/templates/firefox/products/android.html

```diff
@@ -0,0 +1,152 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% from "macros.html" import google_play_button with context %}
+
+{% extends "firefox/products/product-base.html" %}
+
+{# L10n: strings for this page are not considered final and aren't read for localization. #}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{ _('Fast, Free Android Browser for Tablets and Phones | Firefox') }}{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+{% block page_desc %}{{ _('Choose a better browser for your tablet and phone. Firefox is fast and private. Install Firefox for Android now!') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/products/android/product-android-meta.jpg') }}{% endblock %}
+
+{% block body_id %}firefox-product-android{% endblock %}
+
+{% block product_header_content %}
+  <h1 class="page-title"><span>{{ _('Firefox for Android') }}</span></h1>
+  <p class="tagline">{{ _('Our most customizable Firefox for Android yet.') }}</p>
+{% endblock %}
+
+{% block product_header_download %}
+  {% if LANG.startswith('en-') %}
+  <a class="dl-button google-play" rel="external noopener noreferrer" href="{{ alt_href|default(settings.GOOGLE_PLAY_FIREFOX_LINK, True) }}" data-link-type="download" data-download-os="Android" data-mozillaonline-link="{{ settings.GOOGLE_PLAY_FIREFOX_LINK_MOZILLAONLINE }}">
```

Comment:
This was just copied from the macro, maybe the macro doesn't need it either... unless on some pages it *does* open a new tab. Will remove it here.